EU4toV2/Source/EU4World/Modifiers/Modifiers.h

```diff
@@ -1,28 +1,26 @@
 #ifndef MODIFIERS_H
 #define MODIFIERS_H
-
 #include "Modifier.h"
 #include "Parser.h"
 #include <map>
 
 namespace EU4
 {
-	class Modifiers: commonItems::parser
-	{
-	public:
-		Modifiers();
-		explicit Modifiers(std::istream& theStream);
-		
-		void initialize();
-		
-		[[nodiscard]] std::optional<Modifier> getModifier(const std::string& modifierName) const;
+class Modifiers: commonItems::parser
+{
+  public:
+	Modifiers() = default;
+	explicit Modifiers(std::istream& theStream);
+
+	void initialize();
```

Comment:
It's discouraged to have a design that can leave you with an uninitialized copy of a class. This should be moved into the constructor. If that causes problems for the timing of class creation, make the holder have a unique_ptr<Modifiers> and initialize it when you choose.

Reply:
Yeah, lots of old code lying about that I should refactor more thoroughly.